app/views/settings/_theme.haml

```diff
@@ -0,0 +1,135 @@
+      .row
+        .col-sm-6
+          = f.text_field :raised_background, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-6
+          = f.text_field :raised_accent, class: 'color', data: {default: 0xF7F7F7}
+    .card-footer
+      %p Some text on top of a accented area on a raised element!
+  .card
+    .card-body
+      %h2 Colors
+      %p
+        Colors you can find used on the site. The names of the colors usually describe the context.
+        The "text" colors for all styles represent the color that text on top of these colors has.
+
+      %ul
+        %li
+          %b Primary:
+          The primary/brand color of the site, used for navigation, links, etc.
+        %li
+          %b Danger:
+          Color used for errors or critical actions like deleting something.
+        %li
+          %b Warning:
+          Color used for warnings if something non-critical has happened.
+        %li
+          %b Info:
+          Color used for informational popups or messages.
+        %li
+          %b Success:
+          Color used for messages if something went through successfully.
+      .row
+        .col-sm-6
+          = f.text_field :primary_color, class: 'color', data: {default: 0x5E35B1}
+        .col-sm-6
+          = f.text_field :primary_text, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-12
+          .alert.alert-primary
+            A simple primary alert—check it out!
+      .row
+        .col-sm-6
+          = f.text_field :danger_color, class: 'color', data: {default: 0xDC3545}
+        .col-sm-6
+          = f.text_field :danger_text, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-12
+          .alert.alert-danger
+            A simple danger alert—check it out!
+      .row
+        .col-sm-6
+          = f.text_field :warning_color, class: 'color', data: {default: 0xFFC107}
+        .col-sm-6
+          = f.text_field :warning_text, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-12
+          .alert.alert-warning
+            A simple warning alert—check it out!
+      .row
+        .col-sm-6
+          = f.text_field :info_color, class: 'color', data: {default: 0x17A2B8}
+        .col-sm-6
+          = f.text_field :info_text, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-12
+          .alert.alert-info
+            A simple info alert—check it out!
+      .row
+        .col-sm-6
+          = f.text_field :success_color, class: 'color', data: {default: 0x28A745}
+        .col-sm-6
+          = f.text_field :success_text, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-12
+          .alert.alert-success
+            A simple success alert—check it out!
+      .row
+        .col-sm-6
+          = f.text_field :dark_color, class: 'color', data: {default: 0x343A40}
+        .col-sm-6
+          = f.text_field :dark_text, class: 'color', data: {default: 0xFFFFFF}
+        .col-sm-12
+          %button.btn.btn-dark.mb-3 A dark button
```

Comment:
These buttons submit the form... is this intentional?

Reply:
Actually no, pretty weird really. I could turn them into links instead then.